### PR TITLE
README: installation instructions++, download pyinstaller, CI

### PR DESCRIPTION
**Pull request by vindarel**
introduce links to the pyinstaller archive and the lisp core binary built with Gitlab CI.

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -53,12 +53,41 @@ open up new links, or switch tabs with the power of fuzzy completion.
 
 * Getting Started
 ** Install Next
-See the [[https://next.atlas.engineer/download][downloads]] page for pre-built binaries. Some operating systems
-provide packages for Next:
+
+Next is made of two components: a  _lisp core_ and a so called _platform
```
Comment (Ambrevar) [inline README.org]: lisp -> Lisp.

File: README.org [hunk 2]
```diff
@@ -53,12 +53,41 @@ open up new links, or switch tabs with the power of fuzzy completion.
...
+port_. In addition, the platform port comes in two flavours:
+
+- Gtk/[[https://en.wikipedia.org/wiki/WebKit][Webkit]], the browser  engine of the Gnome platform,  also used in
```
Comment (Ambrevar) [inline README.org]: Webkit -> WebKit
(Also mind the spacing.)

Comment (Ambrevar) [reply README.org]: It's WebKitGTK and it's not really the browser engine of the GNOME platform (spelled in all-caps).

File: README.org [hunk 3]
```diff
@@ -53,12 +53,41 @@ open up new links, or switch tabs with the power of fuzzy completion.
...
+  Apple's Safari
```
Comment (Ambrevar) [inline README.org]: Safari uses a different flavour of WebKit, this could be misleading.

Comment (Ambrevar) [reply README.org]: I think Wikipedia links suffice, no need to explain what those renderers are.

File: README.org [hunk 4]
```diff
@@ -53,12 +53,41 @@ open up new links, or switch tabs with the power of fuzzy completion.
...
+- Qt/[[https://en.wikipedia.org/wiki/Blink_(browser_engine)][Webengine/Blink]], the browser engine of Google Chrome.
+
+We  support *GNU/Linux*,  *MacOS*  and *GuixSD*.   Windows support  is
```
Comment (Ambrevar) [inline README.org]: It's `macOS` these days :p

Comment (Ambrevar) [reply README.org]: And it's `Guix` alone, the GuixSD name has been dropped some time ago.

File: README.org [hunk 5]
```diff
@@ -53,12 +53,41 @@ open up new links, or switch tabs with the power of fuzzy completion.
...
+possible but untested.
+
+The recommended way  is to download a *self-contained  Guix archive*. It
+contains the  lisp core,  the Gtk  platform port  and /all  the system
```
Comment (Ambrevar) [inline README.org]: Lisp, GTK

File: README.org [hunk 6]
```diff
@@ -53,12 +53,41 @@ open up new links, or switch tabs with the power of fuzzy completion.
...
+dependencies/. You only have to download and run it !
```
Comment (Ambrevar) [inline README.org]: No space before `!`.

File: README.org [hunk 7]
```diff
@@ -53,12 +53,41 @@ open up new links, or switch tabs with the power of fuzzy completion.
...
+- [[https://guix.gnu.org][Guix]]: Install  with =guix package --install  sbcl-next=
+
+If you want to use the Qt Webengine/Blink platform port, you currently
+have to download it separately.
+
+_Warning_: the Qt platform port is still beta software
+
+- download a  Qt Webengine archive  [[https://next.atlas.engineer/static/release/next-pyqt-webengine.tar.gz][here]]. Once you have  extracted it,
+  run  the  =next-pyqt-webengine= binary.
+- download a GNU/Linux Debian binary of the lisp core [[https://gitlab.com/atlas-engineer/next/pipelines?scope=all&page=1][on our CI system
```
Comment (Ambrevar) [inline README.org]: Lisp

File: README.org [hunk 8]
```diff
@@ -53,12 +53,41 @@ open up new links, or switch tabs with the power of fuzzy completion.
...
+  here]] (you  have to download  the build  artifacts), and run  it with
+  =./next=. You should see a Next window.
```
Comment (Ambrevar) [inline README.org]: I would make all those items proper sentences: start with a capital, ends with a period.

**Comment by vindarel** [comment]
\+1 all fixed and pushed to master.

**Comment by jmercouris** [comment]
I'm sorry I didn't get to review this. In my opinion it is too informative. Someone coming to the GitHub project page should have a simple clear short introduction to the project. Why should they care about any of these details?

**Comment by jmercouris** [comment]
Also, this being merged broke many things within the README, extraneous spaces, lots of triple or double spaces, broken links. Please install a linter in Emacs and look at Magit before pushing.

**Comment by Ambrevar** [comment]
I also think it's too verbose.

**Comment by vindarel** [comment]
> extraneous spaces, lots of triple or double spaces, broken links.

that's very surprising, I don't see them on magit, and I kill save automatically on save.

Ok, so let's improve upon it.

The website should be very user-friendly, I think here we can have some details.

(We have to mention GTK, WebKit, Qt, Blink in the readme!)

**Comment by jmercouris** [comment]
I've significantly trimmed down the readme and mentioned them

**Comment by jmercouris** [comment]
See commit: f94c577 for an example of removing spaces.

**Pull request closed without merging**